docker.nix

```
@@ -125,6 +125,7 @@ let
     sandbox = "false";
     build-users-group = "nixbld";
     trusted-public-keys = "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=";
+    ignored-acls = security.csm;
```

Is this still necessary, now that it's been added as a default to the `ingored-acls` setting? If so, there is no `security` attribute in scope, so it should probably be stringified:
```suggestion
    ignored-acls = "security.csm";
```